catalog/transformer/catalog-transformer-csv-queryresponse/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -31,4 +35,16 @@
         </service-properties>
     </service>
 
+    <service ref="csvMetacardTransformer" interface="ddf.catalog.transform.MetacardTransformer">
+        <service-properties>
+            <entry key="id" value="csv"/>
+            <entry key="mime-type" value="text/csv"/>
+            <entry key="title" value="View as CSV..."/>
+            <!-- deprecated -->
```

Comment:
❓ Are these deprecated entries still required? In general, we shouldn't use deprecated things in new code